video_prediction/lstm_ops.py

```diff
@@ -38,15 +39,12 @@ def init_state(inputs,
   if inputs is not None:
     # Handle both the dynamic shape as well as the inferred shape.
     inferred_batch_size = inputs.get_shape().with_rank_at_least(1)[0]
-    batch_size = tf.shape(inputs)[0]
+    batch_size = inputs.get_shape().as_list()[0]
     dtype = inputs.dtype
   else:
     inferred_batch_size = 0
     batch_size = 0
-
-  initial_state = state_initializer(
-      tf.stack([batch_size] + state_shape),
-      dtype=dtype)
+  initial_state=tf.Variable(tf.zeros(tf.stack([batch_size] + state_shape)))
```

Comment:
I also found out that line 47 should be changed to `initial_state = tf.Variable(tf.zeros([batch_size] + state_shape))`